From fe60c6174563dcf2e193726aac960713c1c522c7 Mon Sep 17 00:00:00 2001
From: Max Horn <max@quendi.de>
Date: Tue, 13 Sep 2022 16:07:40 +0200
Subject: [PATCH 1/3] Don't strip binaries when installing them

This works around an error when cross-compiling on Yggdrasil for macOS:
the `/usr/bin/install` binary is from BusyBox and insists on adding `-p`
to the command line flags when invoking `strip`, but that option is not
supported by the `strip` executable for macOS.
---
 Makefile.rules | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile.rules b/Makefile.rules
index dc619819f..2db76b410 100644
--- a/Makefile.rules
+++ b/Makefile.rules
@@ -583,7 +583,7 @@ install-bin: gap
 	@echo "Warning, 'make install-bin' is incomplete"
 	# install the real GAP executable as $(libdir)/gap/gap
 	$(INSTALL) -d -m 0755 $(DESTDIR)$(libdir)/gap
-	$(INSTALL) -m 0755 -s gap $(DESTDIR)$(libdir)/gap
+	$(INSTALL) -m 0755 gap $(DESTDIR)$(libdir)/gap
 	
 	# install a wrapper shell script invoking the real GAP executable as $(bindir)/gap
 	$(INSTALL) -d -m 0755 $(DESTDIR)$(bindir)
@@ -661,7 +661,7 @@ install-headers: $(FFDATA_H) build/version.h
 install-libgap: libgap.la
 	@echo "Warning, 'make install-libgap' is incomplete"
 	$(INSTALL) -d -m 0755 $(DESTDIR)$(libdir)
-	$(LTINSTALL) -s libgap.la $(DESTDIR)$(libdir)
+	$(LTINSTALL) libgap.la $(DESTDIR)$(libdir)
 
 
 .PHONY: install install-bin install-doc install-gaproot install-headers install-libgap
-- 
2.37.3

